src/Migration/Resources/Database/Column.php

```diff
@@ -55,7 +57,7 @@ public function jsonSerialize(): array
     {
         return [
             'key' => $this->key,
-            'collection' => $this->collection,
+            'table' => $this->table,
             'type' => $this->getType(),
```

Comment:
We can't change the serialized structure, it will break backup restoration

Reply:
Done.